schema.SQL

```diff
@@ -1,43 +1,47 @@
+-- psql command to run this file in terminal:
+-- \i /Users/noahbeito/Desktop/RFP2303/QuestionsAPI/schema.sql
+
 DROP TABLE IF EXISTS questions CASCADE;
 DROP TABLE IF EXISTS answers CASCADE;
 DROP TABLE IF EXISTS photos CASCADE;
 
-CREATE TABLE Questions (
+
+CREATE TABLE questions (
  id BIGSERIAL NOT NULL,
  product_id INTEGER NOT NULL,
  body TEXT NOT NULL,
- date_written TEXT NOT NULL,
+ date_written BIGINT NOT NULL,
  asker_name TEXT NOT NULL,
  asker_email TEXT,
  reported BOOLEAN,
  helpful INTEGER
 );
 
 
-ALTER TABLE Questions ADD CONSTRAINT Questions_pkey PRIMARY KEY (id);
+ALTER TABLE questions ADD CONSTRAINT questions_pkey PRIMARY KEY (id);
 
-CREATE TABLE Answers (
+CREATE TABLE answers (
  id BIGSERIAL NOT NULL,
  question_id INTEGER NOT NULL,
  body TEXT NOT NULL,
- date_written TEXT NOT NULL,
+ date_written BIGINT NOT NULL,
  answerer_name TEXT NOT NULL,
  answerer_email TEXT,
  reported BOOLEAN,
  helpful INTEGER
 );
 
 
-ALTER TABLE Answers ADD CONSTRAINT Answers_pkey PRIMARY KEY (id);
+ALTER TABLE answers ADD CONSTRAINT answers_pkey PRIMARY KEY (id);
 
 CREATE TABLE photos (
  id BIGSERIAL NOT NULL,
- url TEXT NOT NULL,
- answer_id INTEGER NOT NULL
+ answer_id INTEGER NOT NULL,
+ url TEXT NOT NULL
 );
 
 
 ALTER TABLE photos ADD CONSTRAINT photos_pkey PRIMARY KEY (id);
 
-ALTER TABLE Answers ADD CONSTRAINT Answers_question_id_fkey FOREIGN KEY (question_id) REFERENCES Questions(id);
-ALTER TABLE photos ADD CONSTRAINT photos_answer_id_fkey FOREIGN KEY (answer_id) REFERENCES Answers(id);
\ No newline at end of file
+ALTER TABLE answers ADD CONSTRAINT Answers_question_id_fkey FOREIGN KEY (question_id) REFERENCES questions(id);
```

Comment:
This looks like a good strategy to incorporate foreign keys. Did it give you any problems when importing?

Reply:
No issues with importing. I made sure to import questions first, then answers, then photos as each relies on the former for its foreign key (expect questions which does not have a foreign key). I didn't try different import orders, but I assume the order of importing the table data would matter.